clouds/clouds-azure/src/test/java/azure/armTemplates/AzureTestConstants.java

```diff
@@ -6,4 +6,8 @@ public class AzureTestConstants {
     public static final String VNET_NAME_1 = "sunstoneVnet1";
 
     public static final String VNET_NAME_2 = "sunstoneVnet2";
+
+    public static final String IMAGE_REF = "${image.ref:/subscriptions/7dee6f21-9f05-414e-99fa-08d3215fb420/resourceGroups/istraka-test/providers/Microsoft.Compute/images/eap-test-image}";
```

Comment:
that default is allright?

Reply:
This one is OK for now. It is my test image, and I don't have a strategy how to handle it, yet.

Reply:
strategy: something released as default (or first version which will be released) and development image provided as parameter?

Reply:
Yeah, something like that. Or some test image living in our subscription.